### ISSUE TITLE
Final hit in combo should have higher range/crit damage

### ISSUE THREAD

**Issue by MangoLassie142** [ZsoltMolnarrr/BetterCombat#391]
The animation for the final hit is a thrust, so I think it would be fun if it had slightly extended range but a narrower arc. It would give a bit more room for strategy when fighting groups of enemies.

It's also a lot harder to time attacks with jumps for crit damage than in vanilla, so using the combo to get a guaranteed crit at the end would be fun.

**Comment by ZsoltMolnarrr** [comment]
Guaranteed hit is already sort of implemented, as attack specific damage multipliers exists and are used (for example Claymores).
I will consider the attack range multiplier aswell, but I am a little sceptical at the moment.

**Comment by MangoLassie142** [comment]
I think the attack range in general is an issue. Comparing vanilla to BC, you just straight up lose a block of range with BC, which can make a huge difference in fighting something like a Wither Skeleton or anything that's tough to get close to.

**Comment by ZsoltMolnarrr** [comment]
Yes attack range has intentionally been reduced, in an effort to create weapons with varying characteristics.
If you dont like the values, you can override them, using a datapack.

**Comment by MangoLassie142** [comment]
when you say with a datapack, is there one i don't know about? or are you saying I'd have to make my own? Honestly I think there needs to be a toggle in the config for using vanilla range values if you aren't using an expanded weapons mod.
